File: DSA in python/sum.py

def sum(array,target):
    global j
    if len(array)==0:
        print("it is an empty arrray")
    else:
        for i in range(len(array)):
            for j in range(1,len(array)):
                if target == array[i]+array[j]:
                      break
                else:
                    print("not found")
        print("indexes are:",i,j)                        
sum([2,3],5)  

File: DSA in java/sum.java
public class sum {
    static void mysum(int[] array,int target){
        if(array.length==0){
            System.out.println("empty array");
        }
        else{
           for(int i=0;i<array.length;i++){
              for(int j=i+1;j<array.length;j++){
                if(target==array[i]+array[j]){
                    System.out.println("["+i+"," +j+"]");
                }
              }
      }
             
    } 
        }
       
    public static void main(String[] args){
       int[] array={2,4,5};
       int target =5;
       mysum(array, target); 

      }
}